test/frontend-integration-test/helloworld.spec.js

```diff
@@ -135,31 +140,35 @@ describe('deploy helloworld sample run', () => {
     assert(attempts, 'waited for 30 seconds but run did not start.');
 
     assert.equal($$('.tableRow').length, 1, 'should only show one run');
-
-    // Navigate to details of the deployed run by clicking its anchor element
-    browser.execute('document.querySelector(".tableRow a").click()');
-  });
-
-  it('switches to config tab', () => {
-    $('button=Config').waitForVisible(waitTimeout);
-    $('button=Config').click();
   });
 
+  // Wait for a reasonable amount of time until the run is done
+  // and navigate to the run details page
   it('waits for run to finish', () => {
-    let status = getValueFromDetailsTable('Status');
-
     let attempts = 0;
-    const maxAttempts = 60;
+    const maxAttempts = 90;
 
-    // Wait for a reasonable amount of time until the run is done
-    while (attempts < maxAttempts && status.trim() !== 'Succeeded') {
+    while (attempts < maxAttempts) {
```

Comment:
Without the other condition, we are simply pausing 90 seconds before entering the run detail page followed by opening the "Config" tab. So this chunk of change is equivalent to adding a `browser.pause(90 * 1000);` after line 137 (old) / 142 (new). 
Also in my local testing, the "Config" tab for v1 does auto-refresh itself--the "Details" tab for v2 run however doesn't auto-refresh until I click into some blank area on the page. 
So I'm not confident this would solve the test failure in your other PR. 
Can we revert the change in this file and merge only the enabling remote-run change? You can have the one line change of  adding `browser.pause(90 * 1000);` in your other PR to see if it helps.

Reply:
Good point. Reverted.